extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -107,6 +108,7 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 						}
 					});
 				}
+				TelemetryReporter.createActionEvent(BookTelemetryView, NbTelemetryActions.TrustNotebook).send();
```

Comment:
If you're not adding extra properties just call sendActionEvent directly here and all the other similar places